partcad/src/partcad/lint/lint.py

```
@@ -0,0 +1,107 @@
+        }
+
+    @staticmethod
+    def from_dict(data: dict, from_cache: bool = True) -> "LintingReport":
+        report = LintingReport(data["package"])
+        report.cached = from_cache
+        for level, message in data["messages"]:
+            report.add(Severity(level), message)
+        return report
+
+
+class Linting(ABC):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)